HousingSearchListener.Tests/V1/E2ETests/Fixtures/ContractApiFixture .cs

```diff
@@ -29,10 +29,12 @@ public void GivenTheContractDoesNotExist(Guid id)
             // Nothing to do here
         }
 
-        public Contract GivenTheContractExists(Guid id)
+        public Contract GivenTheContractExists(Guid contractId, Guid targetId)
         {
             ResponseObject = _fixture.Build<Contract>()
-                                     .With(x => x.Id, id.ToString())
+                                     .With(x => x.Id, contractId.ToString())
+                                     .With(x => x.TargetId, targetId.ToString())
+                                     .With(x => x.TargetType, "asset")
```

Comment:
For consistency would it be worth introducing Enum for the `TargetType` used in the `Contract`? Currently this is stored as "asset" like here, however when `TargetType` is used in other objects it's typically in a format like `Tenure` . Enum would ensure consistent format of the value across the objects.

Reply:
After discussing and investigation the change touches a lot of different place in the code base and requires it's own spike to investigate the impact.